server/test/fast-food-order.spec.js

```diff
@@ -0,0 +1,25 @@
+import supertest from 'supertest';
+import assert from 'assert';
+import app from '../../app';
+
+const request = supertest(app);
+const path = '/api/v1/orders';
+
+describe('Get all Orders', () => {
+    describe('when Admin request for All fast food order', () => {
+        it('should return all fast food orders', (done) => {
+            request.get(path).end((req, res) => {
+                assert.equal(res.status, 200);
+                done();
+            });
+        });
+    });
+    describe('when Admin request for All fast food order', () => {
+        it('it should have return type of object', (done) => {
+            request.get(path).end((req, res) => {
+                assert.equal(typeof res.body, 'object');
```

Comment:
This assertion is not enough for this testcase. Any object can be returned, you have to check the content of the object.